frontend/lib/src/dataframes/arrowFormatUtils.test.ts

```diff
@@ -14,11 +14,10 @@
  * limitations under the License.
  */
 
-import { vectorFromArray } from "apache-arrow"
+import { Field, Timestamp, TimeUnit, vectorFromArray } from "apache-arrow"
 
 import { Quiver } from "@streamlit/lib/src/dataframes/Quiver"
 import {
-  CATEGORICAL_INTERVAL,
```

Comment:
We can't easily support correctly formatting categorical interval values without relying on Pandas metadata. But is an extreme niche case, and I don't expect this to impact anyone.